### ISSUE TITLE
Compatibility issues between vue.js and GitHub Pages. (Unbuilt static pages)

### ISSUE THREAD

**Issue by binn** [KibiiTV/otako.moe#11]
The current build sitting in the master branch at the moment is **not** compatible with vue.js; we need to separate a branch for debugging releases launched on GitHub Pages.

All debug-deploy (staging) testing and applications are to be deployed to the `gh-pages` branch. After a build of the static files, throw in a PR, and we'll merge into a folder. Examples:

`staging.otako.moe/1.01/index.html`

The static files pulled with your version will be thrown into a version folder (along with a `creator_info.json` and `CREATORINFO.md` file).

Anything on this repository's `gh-pages` branch will be accessible over the internet using `staging.otako.moe` where staging shows that this is an indev open source product. The default *index.html* file will automatically redirect the user to the latest version of our staging. 

If needed, we'll create a simple script to update our staging frontend every time the master branch is updated.

I'd like to request that @PryDt builds version `0.01` of Otako's staging and open's a PR to `gh-pages`. @apap04 and I will take care of merges.

Also yes, we're moving this to `staging.otako.moe` instead of `otako.moe`. Our deployed frontend (`otako.moe`) will be hosted and served statically either from
1. Kibii's CDN, filebin.
2. Our own static servlet.
3. Otako's servlet API.
4. GitHub Pages.

This topic is still undergoing discussion, we'll post more in the README.md file if needed.

That's all this issue has to say, if anyone's wondering, Otako's documentation and specs will be released alongside the [otako.docs](http://github.com/kibiitv/otako.docs/) repo explaining how this will work, and how front-end developers can get started with the webpack version of [otako.js](http://github.com/kibiitv/otako.js).

~bin

**Comment by binn** [comment]
Would like to see a :+1: if you approve of the `staging.otako.moe` idea.

**Comment by prydt** [comment]
Got it, I can also help with documenting front end stuff

**Comment by Greek** [comment]
👍 gc (google cloud) does this, no prob for `staging.otako.moe`

**Comment by Greek** [comment]
its not like im saying to use gc :feelsgood:

**Comment by apap04** [comment]
how is it a bug

**Comment by binn** [comment]
Right, the `gh-pages` branch has been created, and the DNS record for `staging.otako.moe` has been pointed to it, have fun committing.

**Comment by binuwu** [comment]
🍆 this repo looks hot af makes me feel rly porfessional but whateve rkys github

**Comment by binn** [comment]
@apap04 im not gonna fucking use google cloud

**Comment by binuwu** [comment]
wait ignore tha tlast part

**Comment by MeguminSama** [comment]
i'll do it :^)

**Comment by binuwu** [comment]
o um ok

**Comment by binuwu** [comment]
this is rape

**Comment by MeguminSama** [comment]
nice

**Comment by binuwu** [comment]
ok bye rapist

**Comment by MeguminSama** [comment]
F

**Comment by MeguminSama** [comment]
oh. ok

**Comment by Greek** [comment]
reee

**Comment by Greek** [comment]
ima make a dev branch and a build branch

**Comment by binn** [comment]
no dont wtf

**Comment by binn** [comment]
we don't need them and why would we make em?

**Comment by Greek** [comment]
re e e e e 
cuz to see if the page displays so we don’t accidentally merge and someone says that the app doesn’t work. :^)

**Comment by binn** [comment]
You can only have one github pages site per repo

**Comment by binn** [comment]
No one will accidentally merge, we have version control in the gh-pages branch, we'll rollback and disband the version.

**Comment by Greek** [comment]
hopefully no one will accompany merge. im directing that to you, bin ;)

**Comment by binn** [comment]
No one can merge other than you and me LOL.

**Comment by MeguminSama** [comment]
BRB gonna merge :^)

**Comment by binn** [comment]
ok maybe u too

**Comment by MeguminSama** [comment]
It's joke

**Comment by Greek** [comment]
but still ;)

**Comment by binuwu** [comment]
ok can u not im rly not in the mood for jokes atm

**Comment by binuwu** [comment]
i hate myself

**Comment by MeguminSama** [comment]
I used to hate every fiber of my being

**Comment by binuwu** [comment]
ya i do too i wish i wasn't born rn

**Comment by MeguminSama** [comment]
Why

**Comment by binuwu** [comment]
if i was born jap or ameircan or british or german it woulve been nicer

**Comment by binuwu** [comment]
but out of all things im paki

**Comment by binuwu** [comment]
a terrorist race

**Comment by binuwu** [comment]
fuck that

**Comment by MeguminSama** [comment]
;-; cum 2 England

**Comment by binuwu** [comment]
theres no point in spamming my snapchat my dad took my phone lol

**Comment by MeguminSama** [comment]
>terrorist race
Fuck off

**Comment by binuwu** [comment]
no u

**Comment by binuwu** [comment]
i was born to be a terrorist (paki) might as well die before i kill sum1

**Comment by MeguminSama** [comment]
Why would you call yourself a terrorist

**Comment by binuwu** [comment]
because sum1 convinced me that i am one

**Comment by MeguminSama** [comment]
Except you aren't a fucking terrorist

**Comment by binuwu** [comment]
at first i argued with him but i slowly realised that i am

**Comment by binuwu** [comment]
i terrorize people using words, not guns

**Comment by MeguminSama** [comment]
Fuck off yeah.

**Comment by binuwu** [comment]
ok

**Comment by MeguminSama** [comment]
As far as I'm aware you save that for b1nzy or ppl who fite you

**Comment by binuwu** [comment]
what

**Comment by MeguminSama** [comment]
>terrorise with words
>b1nzy

**Comment by binuwu** [comment]
oh right ur right

**Comment by binuwu** [comment]
i keep teorriing him

**Comment by binuwu** [comment]
makes sense

**Comment by MeguminSama** [comment]
If it's against b1nzy it's good

**Comment by binuwu** [comment]
any terrorism is bad

**Comment by binuwu** [comment]
end it all

**Comment by binuwu** [comment]
i was designed to kill people and i dont want to do that

**Comment by MeguminSama** [comment]
Except it's not terrorism

**Comment by MeguminSama** [comment]
You aren't instilling terror, you are prodding him with a pointy stick

**Comment by binuwu** [comment]
terrorism is to create fear

**Comment by binuwu** [comment]
ok

**Comment by binuwu** [comment]
i do that all the time

**Comment by MeguminSama** [comment]
You think b1nzy fears you?

**Comment by MeguminSama** [comment]
Terrorising and instilling fear are slightly different

**Comment by binuwu** [comment]
ok idc

**Comment by binuwu** [comment]
i cant change my race and nor can u so stop talking to me

**Comment by binuwu** [comment]
we literally shitposted on this important issue

**Comment by MeguminSama** [comment]
You ain't a fucking  terrorist and I don't want you to ever speak like that

**Comment by binuwu** [comment]
ye i am

**Comment by MeguminSama** [comment]
You can't change your race but why should you

**Comment by binuwu** [comment]
so
im not labled as a toerrist everywhere

**Comment by binuwu** [comment]
cos thats who i am

**Comment by MeguminSama** [comment]
You aren't a terrorist though. That's a fact. People are given labels their entire life but it doesn't mean they're true.

**Comment by binuwu** [comment]
ya well im starting to realise i am

**Comment by binuwu** [comment]
id stone a homo to death

**Comment by MeguminSama** [comment]
You wouldn't stone me to death

**Comment by binuwu** [comment]
if you were full homo i'd try to make u go back and if not then i will

**Comment by binuwu** [comment]
i despise gays

**Comment by MeguminSama** [comment]
I'd knock you out before you could stone me /shrug

**Comment by YazawaNic0** [comment]
BIN

**Comment by binuwu** [comment]
true

**Comment by YazawaNic0** [comment]
Do you think I’m a terroist I’d say fucking not so stop calling yourself a fuckung terroist you little shit filled precious nugget

**Comment by binuwu** [comment]
nico we were designed to be terrorists now end yourself before you join them

**Comment by MeguminSama** [comment]
Fuck off I wanna sleep early. You ain't a fucking terrorist ffs

**Comment by binuwu** [comment]
go to sleep

**Comment by YazawaNic0** [comment]
Or I️ swear to god I’ll beat you with the sandal of truth for you to realize you are LITERALLY FINE

**Comment by binuwu** [comment]
what if i beat you with a gun

**Comment by YazawaNic0** [comment]
Hey I’d like to end myself you know that but not for that fucking reason

**Comment by apap04** [comment]
stop

**Comment by YazawaNic0** [comment]
I️ have better reasons to kill myself but terroist is not one and neither is it yours

**Comment by MeguminSama** [comment]
bin I s2g if you call yourself a terrorist one more time I'm gonna be pissed

**Comment by apap04** [comment]
ffs

**Comment by binuwu** [comment]
i dont care

**Comment by binuwu** [comment]
now lave me alone

**Comment by binuwu** [comment]
goodnight

**Comment by binuwu** [comment]
i own the repo u dumbass

**Comment by MeguminSama** [comment]
Not until you realise you aren't a terrorist /shrug

**Comment by apap04** [comment]
https://puu.sh/yw8RU/6fdb298d61.jpg

**Comment by binuwu** [comment]
ok im not a teorrist no wgo to sleep fag

**Comment by MeguminSama** [comment]
Lol locked

**Comment by binuwu** [comment]
dose that work?

**Comment by MeguminSama** [comment]
Yeah but are you just trying to get me to go away🤔

**Comment by apap04** [comment]
![image](https://user-images.githubusercontent.com/17275723/33401534-9d80e11a-d527-11e7-8741-0d390b82138f.png)

**Comment by binuwu** [comment]
yes i am

**Comment by YazawaNic0** [comment]
BIN YOU HAVE LITERALLY GIVEN ME AN ENTIRE LECTURE ABOUT HOW ISLAM IS THE PUREST RELIGON ON EARTH NOW TAKE YOUR OWN MEDICINE AND REALIZE THAT YOY ARENT A FUCKING TERRORIST

**Comment by binuwu** [comment]
yes i am

**Comment by binuwu** [comment]
id stone a gay to death

**Comment by binuwu** [comment]
id kill a girl for sex before marriage

**Comment by MeguminSama** [comment]
Yeah you were saying it was gr8. Why change your mind 4 hrs later

**Comment by binuwu** [comment]
starting to relaise it im basically lowkey mad

**Comment by binuwu** [comment]
beucase i thought about what mark said

**Comment by binuwu** [comment]
and hes right

**Comment by YazawaNic0** [comment]
and? That just means you like being kinky that has nothin to do with the religon that you follow

**Comment by YazawaNic0** [comment]
Oof I️ gotta refresh the page more

**Comment by binuwu** [comment]
does it look lik ei fuking care

**Comment by MeguminSama** [comment]
It's okay to not follow a religion 100% and even then Islam is still a peaceful religion. It's a small group of people who are the bad guya

**Comment by binuwu** [comment]
ya well im starting to realise im starting to heed towards those small groups of people

**Comment by binuwu** [comment]
fuck islam its not real

**Comment by binuwu** [comment]
like mark said

**Comment by binuwu** [comment]
relgion is the cause of all issues

**Comment by YazawaNic0** [comment]
Listen mark is a ethically and morally corrupt homosexual who cannot think straight and you think that you can believe any of the shit that comes from his mouth???

**Comment by YazawaNic0** [comment]
You told me yourself Islam has all the proof in the world you just need to study it

**Comment by MeguminSama** [comment]
You're nowhere near a terrorist. Nothing like one. Guy sounds like an ass. Tell him to fuck off. /shrug

**Comment by binuwu** [comment]
ok im not even gonna respond after this

**Comment by MeguminSama** [comment]
K then im just gonna wait here

**Comment by binuwu** [comment]
idc leave my autistic self alone

**Comment by binuwu** [comment]
ok then you can wait like 3 days

**Comment by MeguminSama** [comment]
Nope. Everyone is on the autism spectrum

**Comment by apap04** [comment]
now, is it ok if i delete these messages?

**Comment by MeguminSama** [comment]
nO

**Comment by MeguminSama** [comment]
They still in discord anyways

**Comment by binuwu** [comment]
wait can u delete all the discord webhook msgs i dont want pixel reading this

**Comment by binuwu** [comment]
and apap delete this too pls

**Comment by binuwu** [comment]
cos like

**Comment by binuwu** [comment]
im just gonna keep it to myself

**Comment by MeguminSama** [comment]
She already saw /shrug

**Comment by binuwu** [comment]
no she didnt

**Comment by binuwu** [comment]
now hurry up

**Comment by binuwu** [comment]
i dont care now purge chat

**Comment by MeguminSama** [comment]
If you're gonna post it to a large server that's what happens. I'll delete the whk messages

**Comment by binuwu** [comment]
use r.danny to remove em ok
i cba with a yandere ready to kill someone for me, to start spamming me ok

**Comment by binuwu** [comment]
hey im avoiding everyone

**Comment by binuwu** [comment]
not jus tu

**Comment by binuwu** [comment]
no lol im just commenting on the repo https://i.imgur.com/pkWpuKc.png

**Comment by binuwu** [comment]
ok im gonna "sleep" too

**Comment by binuwu** [comment]
cya

**Comment by apap04** [comment]
ILL DELETE ALL OF THIS

**Comment by binn** [comment]
someone fucking kill me

**Comment by zaida04** [comment]
no ya dont

**Comment by MercerMrc** [comment]
just order a hitman on yourself :^)

**Comment by binn** [comment]
paypal.me/discordapp donate pls

**Comment by MercerMrc** [comment]
uhhh, why?

**Comment by prydt** [comment]
what even happened on this repo?

> btw I'm Indian...
I'm basically a living stereotype

**Comment by apap04** [comment]
recreating issue.. thanks to @binuwu and @MeguminSama

**Comment by YazawaNic0** [comment]
@PryDt Bin and me are too

**Comment by Greek** [comment]
at this point, we should create this issue again.

**Comment by Greek** [comment]
I know what's up. We're ignoring modules from github for it to work properly.
